src/index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module 'graphql-crunch' {
+  export function crunch(data: object | object[]): any[]
+
+  export function uncrunch(data: any[]): object | object[]
```

Comment:
I’m not sure that `object | object[]` is the best output format here.  Accessing `result.something` will cause a ts exception, at least in `noImplicitAny` mode (hope I remember this correctly as i’m from the phone).